### PR TITLE
[WIP] upload aws image and share it with other team accounts.

### PR DESCRIPTION
**Pull request by amavinag**
Purpose or Intent
=================
  * upload aws image and share it with other team account.
    addresses# RHCFQE-1296

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 8bb05fff0a8410e15a963a91f9641d19837bcce2:

`scripts/template_upload_ec2.py`:
- :bangbang: Line 188:16: **E121** *continuation line under-indented for hanging indent*
- :red_circle: Line 189:5: **F841** *local variable 'output' is assigned to but never used*

Please, rectify these issues :smirk: .

Commit inspection report:

**No commit flaws detected**.

*CFME QE Bot*

File: scripts/template_upload_ec2.py [hunk 1]
```diff
@@ -225,12 +247,16 @@ def run(**kwargs):
      """
     mgmt_sys = cfme_data['management_systems']
```
Comment (psav) [inline scripts/template_upload_ec2.py]: Why not just say
```python
if 'ec2east' in cfme_data['management_systems']:
    ssh_rhevm......
```
seems like you are hardcoding it anyway and only using one, there is no need to loop here.

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by mshriver** [comment]
Closing, taking ownership, and replacing with PR #4112

**Pull request closed without merging**